tests/contrib/botocore/test.py

```diff
@@ -528,6 +528,15 @@ def test_s3_put(self):
             ),
         ]
 
+    @mock_s3
+    def test_s3_put_with_add_span_pointers_false(self):
```

Comment:
Should we add a test to specifically test that there are span links when we don't specify an envar?

Reply:
that already exists right above =]